src/templates/haml/static/tour.html.haml

```diff
@@ -49,7 +50,7 @@
         %h1= l('Study our live charts')
         %p= l('Frame your trading strategy using [_1]\'s powerful live charts. Instantly view the trading patterns of forex, indices, commodities and stocks.', stash->{website_name})
         %a.pjaxload.button{:href => url_for('/charting/livechart')}
-          %img.responsive#new-trader-icon{:src => url_for('images/pages/tour/live-chart-snapshot.svg')}
+          %img.responsive#new-trader-icon{:src => url_for('images/pages/tour/live-chart-jp-small.png')}
```

Comment:
@jy-binary sorry, did you mean the standard for file naming is wrong? because that's the name of the image we have newly added with Manros's help for our live chart in tour page